### DISCUSSION TITLE
How is the documentation site built?

### DISCUSSION THREAD

**Discussion by cording12** [vercel/turborepo#8518] [Help] [answered]
### Summary

I really love the docs site for Turbo.build. 

I'm trying to build my own and have been using Mintlify, but not enjoying the experience at the moment. Would love to know how the docs site is built if anyone can offer any ideas?

**Comment by anthonyshew** [comment, marked as answer]
https://fumadocs.vercel.app

**Comment by cording12** [comment]
Perfect - thank you! A follow-up question; in the create-turbo starter, there is a docs and a web folder, each two separate Next.js applications. If I wanted to deploy both on the same domain, so web sits at example.com and docs sits and example.com/docs - is that possible? Would it be a Vercel configuration or something more in-depth?

Edit: Think I found what I'm looking for: https://github.com/vercel/vercel/discussions/8662
